README.md

```diff
@@ -54,28 +56,37 @@ It can:
 Only one address can have this role.
 
 It can:
+
 - Do what the allocators can do.
-- [Timelocked] Enable or disable a market by setting a cap to a specific market.
-    - The cap must be set to 0 to disable the market.
-	- Disabling a market can then only be done if the vault has no liquidity supplied on the market.
+- Decrease the supply cap of any market.
+  - To softly remove a market, it is expected from the allocator role to reallocate the supplied liquidity to another enabled market first.
+- [Timelocked] Increase the supply cap of any market.
+- [Timelocked] Submits the forced removal of a market.
+  - This action is typically designed to force the removal of a market that keeps reverting thus locks the vault.
+  - After the timelock has elapsed, the allocator role is free to remove the market from the withdraw queue. The funds supplied to this market will be lost.
+  - If the market ever functions again, the allocator role can withdraw the funds that were previously lost.
```

Comment:
In the case it has not been removed, right?

Reply:
No, the allocator can always withdraw liquidity from non-enabled markets